integreat_cms/cms/templates/authentication/delete_totp.html

```diff
@@ -0,0 +1,20 @@
+{% extends "authentication/_base.html" %}
+{% load i18n %}
+{% block heading %}
+    {% trans "Add Authentication-App" %}
+{% endblock heading %}
+{% block content %}
+    <div class="mb-3">
+        {% trans "You are about to disconnect your authenticator app from your account and deactivate TOTP authentication. Are you sure about that?" %}
+    </div>
+    <form method="post" action="{% url 'delete_totp' %}">
```

Comment:
```suggestion
    <form method="post">
```
This action also has to be removed to allow non-staff users to disconnect their apps.